Enabling all signature algorithms by default.

In [ssl/ssl_local.h](https://github.com/open-quantum-safe/openssl/blob/OQS-OpenSSL_1_1_1-stable/ssl/ssl_local.h#L207), there are `#define`s that assign one-hot encodings to each signature algorithm. These encodings are used in [ssl/ssl_cert_table.h](https://github.com/open-quantum-safe/openssl/blob/OQS-OpenSSL_1_1_1-stable/ssl/ssl_cert_table.h#L26), and are assigned to a uint32_t variable called `amask` [here](https://github.com/open-quantum-safe/openssl/blob/OQS-OpenSSL_1_1_1-stable/ssl/ssl_local.h#L1924).

Since the encodings are one-hot, at most 32 different signature algorithms can be encoded, and since 8 bits are already used up by OpenSSL by default, at most 24 different OQS signature algorithms can be encoded and assigned to `amask` at any given instant. 

This (possibly?) prevents us from enabling all signature algorithms by default, and the question is how we should deal with this.

After having just enabled all algs, this is the (not entirely surprising) response by a self-respecting compiler (OSX):

```
In file included from ssl/ssl_cert.c:24:
ssl/ssl_cert_table.h:100:45: error: integer literal is too large to be represented in any integer type
    {EVP_PKEY_P384_SPHINCSHARAKA192FSIMPLE, SSL_aP384_SPHINCSHARAKA192FSIMPLE}, /* SSL_PKEY_P384_SPHINCSHARAKA192FSIMPLE */
                                            ^
ssl/ssl_local.h:355:43: note: expanded from macro 'SSL_aP384_SPHINCSHARAKA192FSIMPLE'
#define SSL_aP384_SPHINCSHARAKA192FSIMPLE 0x400000000000000000000U
```
--> We'd need to "widen" integers beyond 64 bits, it seems. Doesn't feel right. 

What'd be the rationale behind including all algs? Do we _really_ need _all_? The count of Sphincs' algs alone made me dizzy....

--- a/oqs-scripts/release-test.sh
+++ b/oqs-scripts/release-test.sh
@@ -1,0 +1,28 @@
+#!/bin/bash
+
+# To be run as part of a release test only on Linux
+
+# must be run in main folder
+# on a serious machine (48+ cores, say)
+
+if [ -d oqs-scripts ]; then
+    # just a temp setup
+    git checkout -b reltest
+    sed -i "s/enable\: false/enable\: true/g" oqs-template/generate.yml && \
+    python3 oqs-template/generate.py && \
+    oqs-scripts/clone_liboqs.sh && \
+    oqs-scripts/build_liboqs.sh && \
+    ./Configure no-shared linux-x86_64 -lm  && make generate_crypto_objects && \
+    make -j 48 && make test && \
+    python3 -m pytest --numprocesses=auto oqs-test/test_tls_full.py oqs-test/test_cms.py oqs-test/test_speed.py && \
+    LIBOQS_LIBTYPE=shared oqs-scripts/build_liboqs.sh && \
+    cp oqs/lib/*.so* . && \
+    make clean && ./Configure shared linux-x86_64 -lm && \
+    make -j 48 && LD_LIBRARY_PATH=. make test && \
+    LD_LIBRARY_PATH=. python3 -m pytest --numprocesses=auto oqs-test/test_tls_full.py oqs-test/test_cms.py oqs-test/test_speed.py
+    # revert temp setup
+    git reset --hard
+else
+    echo "$0 must be run in main oqs-openssl folder. Exiting."
+fi
+

